pkg/pillar/pubsub/driver.go

```diff
@@ -52,7 +52,7 @@ type DriverPublisher interface {
 	Restart(restarted bool) error
 }
 
-// Restarted interface that lets you determine if a Publication has been restarted
+// Restarted interface that lets you determine if a NewPublication has been restarted
```

Comment:
Revert. It is the `Publication` that has been restarted; `NewPublication` is just the func call to create the `Publication`